### PR TITLE
Implementation of Publications and Video sections

### PR DESCRIPTION
**Pull request by ajobi**

### CODE REVIEW

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -32,20 +32,30 @@
...
-        <div class="flex-1 mt-20 md:mr-4">
+        <div class="flex-1 md:mr-14">
             <filter-listing-ska />
         </div>
-        <div class="flex-1 mt-20 md:mr-4">
-            <x-heading-section>
-                Videá
-            </x-heading-section>
+        <div class="flex-1">
+            <x-section-videos
```
Comment (igor-kamil) [inline resources/views/home.blade.php:42]: this is fine. but we should clarify, whether they want to display just one video, or more? (as the header is already suggesting + there is definitely enough space in that column for multiple videos/publications). 
however for not let it be as it's.

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -32,20 +32,30 @@
...
+                title="Videá"
+                embedUrl="https://www.youtube.com/embed/NL1XKbI3x68"
+                linkUrl="#"
+                linkTitle="Pozrite si ďalšie videá"
+            />
+            <x-section-publications
```
Comment (igor-kamil) [inline resources/views/home.blade.php:48]: ok. but it seems quite easy also to obtain the [list of documents from issuu directly](https://developer.issuu.com/managing-your-publications/documents/list) and [display their cover images](https://developer.issuu.com/managing-your-publications/faq/#how-do-i-obtain-an-image-url-of-my-document-cover-) ... also for the next step

**Review by igor-kamil — APPROVED** [review]
LGTM 👌

File: resources/views/home.blade.php (hunk 2 above)
Comment (ajobi) [reply resources/views/home.blade.php:48]: Very nice 👍  I had no idea about the API here.

**Pull request merged**